FlightJournal.Web/Properties/AssemblyInfo.cs

```diff
@@ -33,3 +33,5 @@
 // by using the '*' as shown below:
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
+
+[assembly: log4net.Config.XmlConfigurator(ConfigFile = "log4net.config")]
```

Comment:
in general i would prefer the log4net.config file to be included inside the central configuration file. 
This allows for the configuration to have a cleaner flow if being environment specific and getting changed per environment (in this case i assume the azure runtime would not benefit from files on discs e.g. but other targets for logging)